libraries/kotlinandroidexperimentallibrary/src/main/kotlin/com/uber/okbuck/kotlin/KotlinParcelable.kt

```diff
@@ -0,0 +1,7 @@
+package com.uber.okbuck.kotlin
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+class KotlinParcelable(val name: String) : Parcelable
```

Comment:
just make a new class in the `kotlinlibrary` module. There is no need to make a new one just for this purpose

Reply:
I made the new library to verify that the compiler arguments were indeed different with experimental=true specified. Given that, still think I should just make this a class in the existing `kotlinandroidlibrary`?

Reply:
Ya I think having a separate module just for one extension option is not scalable. As we add more extension options, it will get harder to maintain down the line

Reply:
Isn't that the way Buck wants you to structure projects :trollface:

Reply:
Done